doc/src/Howto_moltemplate.rst

```diff
@@ -0,0 +1,371 @@
+Moltemplate Tutorial
+====================
+
+In this tutorial, we are going to use the tool :ref:`Moltemplate
+<moltemplate>` to set up a classical molecular dynamic simulation using
```

Comment:
... use the Moltemplate tool, included in the LAMMPS distribution, to setup a classical ...

Reply:
oops - I didn't read the Tools section carefully enough.  You can ignore my suggested Howto doc change